datastore/datastore.go

```diff
@@ -679,3 +701,40 @@ func (c *Client) Mutate(ctx context.Context, muts ...*Mutation) (ret []*Key, err
 	}
 	return ret, nil
 }
+
+// ReadTime specifies a snapshot (time) of the database to read.
+func ReadTime(t time.Time) ReadOption {
+	return docReadTime{t}
+}
+
+type docReadTime struct {
```

Comment:
If you don't intend to add any additional fields to this struct you could also do this:

``` golang
type docReadTime time.Time
```

Reply:
Done.